### PR TITLE
Kevin yoo/hmp 288 entity api failure messaging for globus links

### PR DESCRIPTION
**Pull request by kevinyooky**

### CODE REVIEW

**Comment by kevinyooky** [comment]
I was cleaning up my branches from my local terminal and had mistaken this with another branch. This branch should NOT have been deteled.

File: context/app/static/js/components/detailPage/BulkDataTransfer/NoAccess.jsx [hunk 1]
```diff
@@ -1,15 +1,29 @@
...
 
 import { DetailSectionPaper } from 'js/shared-styles/surfaces';
-import { StyledWarningIcon, WarningIconContainer, NoAccessContainer } from './style';
+import { SecondaryBackgroundTooltip } from 'js/shared-styles/tooltips';
+import { Header, StyledWarningIcon, IconContainer, NoAccessContainer, StyledExclamationIcon } from './style';
 
-function NoAccess({ children }) {
+const statusIcons = {
+  warning: <StyledWarningIcon />,
+  exclamation: <StyledExclamationIcon color="error" fontSize="small" />,
```
Comment (NickAkhmetov) [inline context/app/static/js/components/detailPage/BulkDataTransfer/NoAccess.jsx:10]: `error` might be a more suitable/clear status name

**Review by NickAkhmetov — COMMENTED** [review]
Thanks for the clarification re: what happened with the other branch!

A few finishing touches:

File: context/app/static/js/components/detailPage/BulkDataTransfer/NoAccess.jsx [hunk 2]
```diff
@@ -1,15 +1,29 @@
...
+      {status === 'exclamation' && (
+        <Header variant="h5">
+          {title}
+          {status ? statusIcons[status] : null}
+          <SecondaryBackgroundTooltip title={tooltip}>
+            <InfoIcon fontSize="small" color="primary" />
+          </SecondaryBackgroundTooltip>
+        </Header>
+      )}
+      <NoAccessContainer displayValue={status === 'exclamation' ? 'display' : 'flex'}>
```
[the remark this answers is not in the document]
Comment (kevinyooky) [reply context/app/static/js/components/detailPage/BulkDataTransfer/NoAccess.jsx:25]: That is a good catch...

**Comment by kevinyooky** [comment]
Thanks Nik for all the comments!

File: context/app/static/js/components/detailPage/BulkDataTransfer/usePanelSet.jsx [hunk 3]
```diff
@@ -180,31 +184,39 @@ const NON_CONSORTIUM_MEMBERS_NO_DBGAP = {
...
-      </EmailIconLink>
-      &nbsp;with the dataset ID and information about the files you are trying to access.
-    </div>
-  ),
+  error: {
+    status: 'error',
+    title: 'HuBMAP Globus Access',
+    tooltip: globusText.tooltip,
+    children: (
+      <>
```
Comment (NickAkhmetov) [inline context/app/static/js/components/detailPage/BulkDataTransfer/usePanelSet.jsx:208]: Out of curiosity - I noticed the `warning` panels are wrapped in a `div`, while this `error` panel is using a fragment. Is this related to the toggle between `block` and `flex`?

**Review by NickAkhmetov — APPROVED** [review]
Looks good to me, great job! I did have a quick question but I don't think it's blocking.

@john-conroy should take a look as well to be safe

File: context/app/static/js/components/detailPage/BulkDataTransfer/usePanelSet.jsx (hunk 3 above)
Comment (kevinyooky) [reply context/app/static/js/components/detailPage/BulkDataTransfer/usePanelSet.jsx:208]: There is no particular reason for this, some of them are divs and some of them are fragments. 

If this is something we want to be particular on, then we would need to decide on which one we want to go with, as they are pretty much being used interchangeably.

**Pull request merged**